src/main/openapi/ds-datahandler-openapi_v1.yaml

```diff
@@ -428,8 +426,11 @@ components:
             * `date`:     Only use format yyyy-MM-dd as from/until parameters
             * `datetime`:    Use UTC format yyyy-MM-ddThh:mm:ssZ as from/until parameters. This is default.                        
           example: 'date'
-          default: 'date'         
-          
+          default: 'date'
+        fragmentServiceUrl:
+          type: string
+          description: BaseURL for the enrichment fragment service
+          example: 'http://example.com/'
```

Comment:
This is indeed the correct way. Could you add an example value for the origin ds.radiotv in ds-datahandler-behaviour.yaml oaiTargets and also in the corresponding aegis file for the devel system?

Reply:
done

Reply:
I can't find the updated version in aegis for the devel system, but I like how its done in the behaviour file!